src/components/Image.astro

```diff
@@ -34,6 +34,14 @@ const {
   ...rest
 } = Astro.props
 
+const isLocalAsset = src && (src.startsWith('*/') || src.startsWith('../'))
```

Comment:
There is a better and more reliable way to check if something is a URL (so not-local). Know know from the top of my head, but look it up

Reply:
```suggestion
const isUrl = (str: string): boolean => {
```